Sources/XcodeProj/Errors/Errors.swift

```diff
@@ -1,5 +1,5 @@
 import Foundation
-import PathKit
+@preconcurrency import PathKit
```

Comment:
I wonder if we should take the opportunity to adopt [Path](https://github.com/tuist/Path/), where we forked `AbsolutePath` and `RelativePath` from swift-tools-support, and use those types instead? The long-term plan is to update Tuist to use that package, so we can start by updating its dependencies.
This is something that we can do in a follow-up PR